packages/calypso-products/src/plans-list.js

```diff
@@ -720,7 +720,10 @@ const getPlanJetpackSecurityRealtimeDetails = () => ( {
 const getPlanJetpackCompleteDetails = () => ( {
 	group: GROUP_JETPACK,
 	type: TYPE_ALL,
-	getTitle: () => translate( 'Complete' ),
+	getTitle: () =>
+		translate( 'Complete', {
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **73** times:
`translate( 'Complete' )` *ES Score: 10*